src/main/java/noobconversation/Conversation.java

```diff
@@ -0,0 +1,74 @@
+package noobconversation;
+
+import format.Formatting;
+import task.Task;
+
+import java.util.Scanner;
+
+public class Conversation {
+
+    protected Formatting f;
```

Comment:
Name can be more descriptive. You have use the name `format` in other classes, you can do the same here.